Userland/Libraries/LibLine/InternalFunctions.cpp

```diff
@@ -569,11 +569,7 @@ void Editor::edit_in_external_editor()
     }
 
     {
-        auto file_or_error = Core::Stream::File::open({ file_path, strlen(file_path) }, Core::Stream::OpenMode::Read);
-        if (file_or_error.is_error())
-            return;
-
-        auto file = file_or_error.release_value();
+        auto file = Core::Stream::File::open({ file_path, strlen(file_path) }, Core::Stream::OpenMode::Read).release_value_but_fixme_should_propagate_errors();
```

Comment:
This changes behaviour - previously an error opening the file would `return`, but now it'll crash. That might be fine, but it's iffy.

Reply:
I considered it fine since we already `release_value_but_fixme_should_propagate_errors` on all other IO errors anyways. Of course, if there is a reasonable chance that the user hits this path specifically during normal operation and "silently exit without indicating any error" is the intended result, I'm happy to change it back.